core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```diff
@@ -165,31 +64,22 @@ class SnapshotServiceSpec extends AnyWordSpecLike with Matchers with MockitoSuga
         .thenReturn(new GcpBigQueryDatasetResource().metadata(new ResourceMetadata().resourceId(UUID.randomUUID())).attributes(new GcpBigQueryDatasetAttributes()))
 
       val workspace = minimalTestData.workspace
-      val deltaLayer = new DeltaLayer(mockBigQueryServiceFactory, new MockDeltaLayerWriter, mockSamDAO, fakeRawlsClientEmail, fakeDeltaLayerStreamerEmail)
 
       val snapshotService = SnapshotService.constructor(
         slickDataSource,
         mockSamDAO,
         mockWorkspaceManagerDAO,
-        deltaLayer,
-        "fake-terra-data-repo-dev",
-        fakeRawlsClientEmail,
-        fakeDeltaLayerStreamerEmail
+        "fake-terra-data-repo-dev"
       )(userInfo)
 
-      val deltaLayerDatasetName = DeltaLayer.generateDatasetNameForWorkspace(workspace)
-
       val snapshotUUID = UUID.randomUUID()
 
       Await.result(snapshotService.deleteSnapshot(workspace.toWorkspaceName, snapshotUUID.toString), Duration.Inf)
 
       verify(mockWorkspaceManagerDAO, times(1)).getDataRepoSnapshotReference(ArgumentMatchers.eq(workspace.workspaceIdAsUUID), ArgumentMatchers.eq(snapshotUUID),any[OAuth2BearerToken])
       verify(mockWorkspaceManagerDAO, times(1)).deleteDataRepoSnapshotReference(ArgumentMatchers.eq(workspace.workspaceIdAsUUID), ArgumentMatchers.eq(snapshotUUID), any[OAuth2BearerToken])
       // assert we do NOT attempt to delete the companion dataset or its WSM reference
-      verify(mockWorkspaceManagerDAO, times(0)).getBigQueryDatasetReferenceByName(ArgumentMatchers.eq(workspace.workspaceIdAsUUID), ArgumentMatchers.eq(deltaLayerDatasetName), any[OAuth2BearerToken])
       verify(mockWorkspaceManagerDAO, times(0)).deleteBigQueryDatasetReference(ArgumentMatchers.eq(workspace.workspaceIdAsUUID), any[UUID], any[OAuth2BearerToken])
```

Comment:
I think that lines 81-82 - the comment and the `verify(mockWorkspaceManagerDAO, times(0)).deleteBigQueryDatasetReference …` - are now obsolete too

Reply:
Agreed! Found a few more obsolete lines off of these too- good catch!